apps/emqx/src/emqx_config.erl

```diff
@@ -280,7 +281,7 @@ init_load(SchemaMod, RawConf) when is_map(RawConf) ->
                             maps:with(RootNames, RawConfWithEnvs)).
 
 include_dirs() ->
-    [filename:join(emqx:data_dir(), "configs")].
+    [filename:join(emqx:data_dir(), "configs") ++ "/"].
```

Comment:
is this still needed ?
e.g. in windows, the path will become `prefix/\name` after `filename:join`

Reply:
OH, forget windows. 
we don't need  this  now !